install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -110,7 +128,11 @@ spec:
           # The initial delay for the liveness probe is intentionally large to
           # avoid an endless kill & restart cycle if in the event that the initial
           # bootstrapping takes longer than expected.
+{{- if semverCompare ">=1.20-0" .Capabilities.KubeVersion.Version }}
+          initialDelaySeconds: 0
```

Comment:
nit: default value of initialDelaySeconds is 0, so we can omit this if you want.

Reply:
Fixed